### PR TITLE
Try to fix regex

### PR DESCRIPTION
**Pull request by pmuir**

### CODE REVIEW

**Comment by pmuir** [comment]
Not ready yet

**Comment by pmuir** [comment]
@paulrobinson ready for review

File: javascripts/terms-and-conditions.js [hunk 1]
```diff
@@ -57,7 +57,7 @@ app.termsAndConditions = {
 
       // Inform GTM that we have requested a product download
       window.dataLayer = window.dataLayer || [];
-      window.dataLayer.push({ 'product_download_file_name' : $.encoder.encodeForJavascript(tcDownloadFileName).replace(/\x2D/g, '-') });
```
Comment (paulrobinson) [inline javascripts/terms-and-conditions.js]: So you no longer need to fix the invalid encoding? Did you fix it elsewhere?

Comment (pmuir) [inline javascripts/terms-and-conditions.js]: It looks like the problem in the end was encoding the tcDownloadFileName again using $.encoder.encodeForJavascript

**Pull request merged**